runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/util/CloudObjectsTest.java

```diff
@@ -112,7 +121,25 @@ public void defaultCodersIncludesCustomCoder() {
                       KvCoder.of(VarLongCoder.of(), ByteArrayCoder.of()),
                       IntervalWindow.getCoder()))
               .add(VarLongCoder.of())
-              .add(ByteArrayCoder.of());
+              .add(ByteArrayCoder.of())
```

Comment:
Can you move ByteArrayCoder.of up so this list and the list in DefaultCoderRegistrars is in the same order, makes it easier to review.